src/argilla_server/apis/v1/handlers/users.py

```diff
@@ -36,6 +36,19 @@ async def get_current_user(request: Request, current_user: models.User = Securit
     return current_user
 
 
+@router.get("/users", response_model=Users)
+async def list_users(
+    *,
+    db: AsyncSession = Depends(get_async_db),
+    current_user: models.User = Security(auth.get_current_user),
+):
+    await authorize(current_user, UserPolicyV1.list)
+
+    users = await accounts.list_users(db)
+
+    return Users(items=users)
```

Comment:
This endpoint will return API keys for all the users. Maybe is safer to exclude this attribute.

Reply:
Do you think we should remove it for all endpoints returning users or only here?

Reply:
Let's discuss this later. For now, it's okay as is.